packages/odie-client/src/index.tsx

```diff
@@ -25,35 +22,9 @@ export const OdieAssistant: React.FC = () => {
 	useAutoScroll( messagesContainerRef, chat.messages );
 	useLastMessageVisibility( messagesContainerRef, chat.messages.length );
 
-	useEffect( () => {
```

Comment:
It is not clear to me what this was doing and why we are removing it. I would just like to understand the changes better. Could you elaborate a bit?

Reply:
This honestly was an intermediate step that I should have deleted with the previous rework. We never have 0 messages as the messages array is always init with Wapuu message, I thought then that the responsibility of showing the spinner is not for this component itself, but for the parent.

Reply:
Just for my better understanding, these changes are not related to the issue, they are more like a refactoring, right?